### PR TITLE
[WIP] Webhooks CRUD

### PR DESCRIPTION
**Pull request by claudiosanches**
This branch create a few databases, so before testing is required to disable and enable WooCommerce again.

### CODE REVIEW

**Comment by claudiulodro** [comment]
Everything seems to be working correctly. Is this ready to merge @claudiosanches?

**Comment by claudiosanches** [comment]
@claudiulodro I just fixed some conflicts and all ready to merge now.

File: includes/wc-update-functions.php [hunk 1]
```diff
@@ -1200,6 +1205,7 @@ function wc_update_300_webhooks() {
 }
 
 /**
+>>>>>>> master
```
Comment (claudiulodro) [inline includes/wc-update-functions.php]: Some unresolved merge conflicts in this file. This looks good to merge once those are fixed.

Comment (claudiosanches) [reply includes/wc-update-functions.php]: Sorry, just fixed it.

**Pull request merged**